pallets/funding/src/traits.rs

```diff
@@ -32,6 +32,22 @@ pub trait ProvideAssetPrice {
 	type AssetId;
 	type Price: FixedPointNumber;
 	fn get_price(asset_id: Self::AssetId) -> Option<Self::Price>;
+
+	fn calculate_decimals_aware_price(
+		original_price: Self::Price,
+		usd_decimals: u8,
+		asset_decimals: u8,
+	) -> Option<Self::Price> {
+		let usd_unit = 10u128.checked_pow(usd_decimals.into())?;
```

Comment:
Should probably be one? Now usd_unit is 10_000_000? 
Same for `asset_unit`.

Reply:
10^6 = 1_000_000
1 ^ 6 = 1

So this is correct, or am I missing something?

Reply:
Oh wow, my math head let me down here. You are completely right!